ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -61,6 +61,39 @@ def msort(a):
     return ivy.msort(a)
 
 
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {
+        "0.4.13 and below": (
+            "float16",
+            "bfloat16",
+        )
+    },
+    "jax",
+)
+def argpartition(
+    a,
+    kth,
+    axis=-1,
+    kind="stable",
+    order=None,
+) -> ivy.Array:
```

Comment:
Correct me if I'm wrong but the signature for `jax.numpy.argpartition` doesn't have `kind` and `order` arguments? [reference](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.argpartition.html)